de.bund.bfr.knime.fsklab.nodes/src/de/bund/bfr/knime/fsklab/nodes/ReaderNodeModel.java

```diff
@@ -77,6 +77,8 @@
 import de.bund.bfr.knime.fsklab.FskPortObjectSpec;
 import de.bund.bfr.knime.fsklab.FskSimulation;
 import de.bund.bfr.knime.fsklab.JoinRelation;
+import de.bund.bfr.knime.fsklab.nodes.JoinerNodeUtil;
+import de.bund.bfr.knime.fsklab.nodes.NodeUtils;
```

Comment:
We should remove the import to NodeUtils since it is not needed because ReaderNodeModel is in the same package.